### ISSUE TITLE
Add new attribute slow_start to aws_lb_target_group

### ISSUE THREAD

**Issue by bboerst** [terraform-providers/terraform-provider-aws#4660]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

AWS recently released an attribute called 'Slow Start' for Application Load Balancers. This new aws_lb_target_group attribute should be added to the provider.

This change requires v1.13.55 of the aws-sdk-go package.

### New or Affected Resource(s)

* aws_lb_target_group

### Potential Terraform Configuration

```hcl
resource "aws_lb_target_group" "test" {
  name     = "tf-example-lb-tg"
  port     = 80
  protocol = "HTTP"
  vpc_id   = "${aws_vpc.main.id}"
  
  deregistration_delay = 300
  slow_start = 30
}
```

### References
* https://aws.amazon.com/about-aws/whats-new/2018/05/application-load-balancer-announces-slow-start-support/
* https://docs.aws.amazon.com/elasticloadbalancing/latest/application/load-balancer-target-groups.html#slow-start-mode

**Comment by bflad** [comment]
This will be released with v1.21.0 of the AWS provider later today.

**Comment by bflad** [comment]
This has been released in [version 1.21.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1210-may-31-2018). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
